src/preprocessing.py

```diff
@@ -15,8 +16,9 @@ def preprocess_data():
     # TODO: Issue of raw path
     data_repo_url = "https://github.com/axondeepseg/data_axondeepseg_sem"
     data_dir = "data_axondeepseg_sem"  # Local directory for the cloned repository
-    processed_images_dir = "processed_images"
-    processed_masks_dir = "processed_masks"
+    processed_images_dir = "dataset/images/train"
+    processed_masks_dir = "dataset/labels/train"
```

Comment:
Reminder to split the data into train-val-test splits.

Reply:
Corrected in the last commit